docs/workflows/mlflow-builder.md

```diff
@@ -0,0 +1,146 @@
+
+
+## Using the MLflow Runtime Environment Step
+
+The container image built by the MLflow builder workflow step can itself be used as a FuseML workflow step. The same codeset used as input for the MLflow builder must be supplied as input to the MLflow runtime environment. The MLflow runtime environment workflow step executes the MLflow code present in the codeset and returns the URL where one or more MLflow artifacts are stored during the execution. Depending on how the MLflow code is configured to run, this can be a local path or a URL pointing to a remote MLflow artifact store.
+
+The MLflow runtime environment container image also accepts the following optional workflow input parameters that can be used to customize how the MLflow code is executed:
+
+- `mlflow_experiment` - the experiment ID to use when running the MLflow code. If not specified, the experiment name is derived from the codeset name and project used as input
+- `mlflow_entrypoint` - controls which of the MLflow entrypoints configured in the `MLproject` file used to build the image is executed. If not specified, `main` is used as the default entrypoint.
+- `mlflow_entrypoint_args` - space or comma-separated list of additional MLflow run entrypoint arguments in the form `name=value`. These arguments are passed to the MLflow entrypoint as keyword arguments. For example, a value of `epochs=2,batch_size=64` will pass the arguments `epochs=2` and `batch_size=64` to the MLflow entrypoint.
+- `mlflow_artifact_path` - sub-path under the MLflow artifact repository that should be returned as output. By default, the `model` sub-directory is returned, because this is where MLflow stores the ML models.
+- `verbose` - set to `true` to enable verbose logging in the MLflow runtime step (default is `false`).
+
+The MLflow runtime workflow step can also take in additional environment variables that are passed transparently to the MLflow runtime and used to configure how the MLflow runtime can access a remote MLflow tracking server and artifact store. For more information on what variables are available and how they can be used, please refer to the [relevant section in the official MLflow documentation](https://www.mlflow.org/docs/latest/tracking.html#logging-to-a-tracking-server). Only a subset of the available variables are listed here:
+
+> :warning: Note that some of these environment variables contain sensitive data, such as keys and passwords and should not be explicitly configured as workflow step env vars. Instead, they should be configured in the FuseML Extension Registry and only referenced in the FuseML workflows as extension requirements.
```

Comment:
This is rendered strangely, I assume you wanted something else...

Reply:
you're right, thanks for noticing. This format works with the github markdown renderer, but it doesn't work here. I fixed it to look like a note section.